Applications/Internet/Internet Explorer 6.0/Online/script.js

```diff
@@ -10,91 +10,159 @@ include("utils.functions.apps.resources");
 include("engines.wine.verbs.msls31");
 
 var installerImplementation = {
```

Comment:
I couldn't change this script to the new style because it doesn't use a quick script. @plata @ImperatorS79 do you have an idea how we could transform this to an JS object?

Reply:
Can't you write it similarly to https://github.com/PhoenicisOrg/scripts/blob/master/Engines/Wine/QuickScript/Installer%20Script/script.js (except that it does not `extend QuickScript`)?

Reply:
Ok I've now introduced a `DummyInstallerScript` class, which is used by the IE installers.

As a note:
Both IE scripts are broken, independent of the changes done in this PR. IE 6 fails when downloading some file. IE 7 fails because of a syntax error in the script. 

The IE 7 syntax error I fixed, but after completing the installation I was unable to start it.

Reply:
What's the advantage of the `DummyInstallerScript` compared to directly implementing the `go` method? Originally, I was thinking of something like:
```Javascript
class Ie6Installer {
    go() {
        ...
    }
}
new Ie6Installer();
```

Reply:
There is not really an advantage. It just makes it simpler to read the file. In addition it is more in line with the style used for the other installers, which only contain a `new XYZInstallerScript()...` statement.

Reply:
Ok. I'm fine with using the approach but I really don't like the name nor the place. `DummyInstaller` somehow suggests that it's not a "real" installer. Also, it can be used for all engines (not just Wine) so it should not be in `Wine/QuickScripts`.

Reply:
I'm not happy with introducing the class at all. In theory the IE installers should be some kind of `OnlineInstallerScript`s because they download an installer. In addition the installers don't work (independent of this PR). The best solution in my eyes would be to rewrite the IE installers to make use of `OnlineInstallerScript`.

Reply:
They are a showcase of how you can write completely free scripts (we explicitly want to support that).

Reply:
I understand, then I think we should go with my `DummyInstallerScript` approach and give it a better name and move it a folder up to make it independent of Wine. 

Still we have the issue that the IE installers don't seem work.

Reply:
How about `Utils/Functions/Apps/Installers` and `PlainInstaller`? I currently do not care if the script works (if it was broken before already, this is not related to this pr).

Reply:
Ok I have renamed `DummyInstallerScript` to `PlainInstaller`